modules.nix

```diff
@@ -55,6 +55,19 @@ let
                   {
                     options =
                       {
+                        deleteOldIfChanged = lib.mkOption
+                          {
+                            type = bool;
+                            default = true;
+                            description = ''
+                              Destroy and recreate old libvirt objects if a NixOS configuration switch changes
+                              the nixvirt service unit (e.g. by defining a new domain).
+
+                              Note that, if enabled, all domains, networks and storage pools will be forcefully
+                              shutdown and restarted upon deploying a new configuration. Otherwise, the same result
```

Comment:
I think this description is incorrect: NixVirt (imperfectly) attempts to determine whether a definition has changed and doesn't deactivate an object if it thinks it hasn't.

Reply:
Agree, this came from my initial misunderstanding of the module helper's behavior. Let me improve that description.